docs/ref/forms/widgets.txt

```diff
@@ -850,6 +850,21 @@ Composite widgets
 
         Similar to :attr:`TimeInput.format`
 
+    .. attribute:: SplitDateTimeWidget.date_attrs
+
+        .. versionadded:: 2.0
+
+        Similar to :attr:`Widget.attrs`. A dictionary containing HTML
+        attributes to be set on the rendered :class:`DateInput` widget.
+
+    .. attribute:: SplitDateTimeWidget.time_attrs
+
+        .. versionadded:: 2.0
+
+        Similar to :attr:`Widget.attrs`. A dictionary containing HTML
+        attributes to be set on the rendered :class:`TimeInput` widget.
+
```

Comment:
I think it's worth mentioning the `attrs` is used for both widgets if these aren't set. Since the arguments are so similar, you could put the `.. attribute::` lines next to each other and have them share a description.